minard/static/js/scatter_plot_rates.js

```diff
@@ -0,0 +1,118 @@
+function history() {
+    var params = {}
+    params['crate'] = document.getElementById("crate-sel").value;
+    params['slot'] = document.getElementById("slot-sel").value;
+    params['channel'] = document.getElementById("channel-sel").value;
+    window.location.replace($SCRIPT_ROOT + "/check_rates_history?" + $.param(params));
+}
+
+var data = window.data;
+if( data != undefined && data != "" ){
+    var data_length = data.length;
+    var last_run = data[0][0];
+    var first_run = data[data_length-1][0]
+    var len = (last_run - first_run)/data.length;
+    var run_offset = 0;
+    var start_offset = 103214;
+    var start = start_offset - run_offset;
+    var end = d3.max(data, function(d) { return d[0]; }) + 1 - run_offset;
+    
+    var margin = {top: 20, right: 15, bottom: 60, left: 260}
+        , width = 1600 - margin.left - margin.right
```

Comment:
For the width here, you can grab the screen width by selecting the div that it's going to be added to and getting the width, i.e.:

```
width = $("#main").width();
```

Currently, it shows up really big on my browser:


![check_rates_history](https://user-images.githubusercontent.com/5834059/28525981-a8b8b314-704b-11e7-842f-36823a80794c.png)